chat-widget/src/common/telemetry/TelemetryConstants.ts

```diff
@@ -251,6 +262,11 @@ export class TelemetryConstants {
             case TelemetryEvent.CustomerVoiceResponsePageLoaded:
             case TelemetryEvent.CustomerVoiceFormResponseSubmitted:
             case TelemetryEvent.CustomerVoiceFormResponseError:
+            case TelemetryEvent.LinkModePostChatWorkflowStarted:
+            case TelemetryEvent.EmbedModePostChatWorkflowStarted:
+            case TelemetryEvent.PostChatWorkflowFromCustomer:
+            case TelemetryEvent.PostChatWorkflowFromAgent:
+            case TelemetryEvent.PostChatWorkflowFromBot:
```

Comment:
Are these additions complete? I see some events from above that are not here

Reply:
Added to Telemetry docs